docs/dev-guide/themes.md

```diff
@@ -116,6 +123,58 @@ themes for consistency.
 [template inheritance]: https://jinja.palletsprojects.com/en/latest/templates/#template-inheritance
 [blocks]: ../user-guide/customizing-your-theme.md#overriding-template-blocks
 
+### Picking up CSS and JavaScript from the config
+
+MkDocs defines the top-level [extra_css](../user-guide/configuration.md#extra_css) and [extra_javascript](../user-guide/configuration.md#extra_javascript) configs. These are lists of files.
+
+The theme must include the HTML that links the items from these configs, otherwise the configs will be non-functional. You can see the recommended way to render both of them in the [base example above](#basic-theme).
+
+> NEW: **Changed in version 1.5:**
+>
+> The items of the `config.extra_javascript` list used to be simple strings but now became objects that have these fields: `path`, `type`, `async`, `defer`.
+>
+> In that version, MkDocs also gained the [`script_tag` filter](#script_tag).
+>
+> >? EXAMPLE: **Obsolete style:**
+> >
+> > ```django
+> >   {%- for path in extra_javascript %}
+> >     <script src="{{ path }}"></script>
+> >   {%- endfor %}
+> > ```
+> >
+> > This old-style example even uses the obsolete top-level `extra_javascript` list. Please always use `config.extra_javascript` instead.
+> >
+> > So, a slightly more modern approach is the following, but it is still obsolete because it ignores the extra attributes of the script:
+> >
+> > ```django
+> >   {%- for path in config.extra_javascript %}
+> >     <script src="{{ path | url }}"></script>
+> >   {%- endfor %}
+> > ```
+> <!-- -->
```

Comment:
Extra HTML comment?

Reply:
This is necessary for old-school Markdown to not merge two blockquotes into one

Reply:
In this particular case, markdown-callouts actually prevents merging these blockquotes, but markdownlint doesn't know that and complains

Reply:
Oh OK, thanks :+1: